### ISSUE TITLE
compile error on intel-apollolake pulsar linux 8

### ISSUE THREAD

**Issue by aaronadlink** [WindRiver-OpenSourceLabs/wr-core#39]
i try get an new bsp source code in the this morning (2017/03/08)
i didn't modify any source code on BSP , configure to init-intel-aollolake-env
compile error on cups package , the error message :

ippfind.c: In function 'main':
| ippfind.c:1367:21: error: 'ippfind_srv_t {aka struct ippfind_srv_s}' has no member named 'ref'
|          if (!service->ref && !service->is_resolved)
|                      ^
| ippfind.c:1416:22: error: 'ippfind_srv_t {aka struct ippfind_srv_s}' has no member named 'ref'
|            if (service->ref)
|                       ^
| ippfind.c:1421:13: warning: implicit declaration of function 'avahi_service_resolver_free' [-Wimplicit-function-declaration]
|              avahi_service_resolver_free(service->ref);
|              ^
| ippfind.c:1421:48: error: 'ippfind_srv_t {aka struct ippfind_srv_s}' has no member named 'ref'
|              avahi_service_resolver_free(service->ref);
|                                                 ^
| ippfind.c:1424:13: error: 'ippfind_srv_t {aka struct ippfind_srv_s}' has no member named 'ref'
|       service->ref = NULL;
|              ^
| ippfind.c:1432:25: error: 'ippfind_srv_t {aka struct ippfind_srv_s}' has no member named 'ref'
|          else if (service->ref)
|                          ^
| ippfind.c: In function 'dnssd_error_string':
| ippfind.c:1732:1: warning: no return statement in function returning non-void [-Wreturn-type]
|  }
|  ^
| ippfind.c: In function 'get_service':
| ippfind.c:2119:18: error: 'kDNSServiceMaxDomainName' undeclared (first use in this function)
|    char  fullName[kDNSServiceMaxDomainName];
|                   ^
| ippfind.c:2119:18: note: each undeclared identifier is reported only once for each function it appears in
| ippfind.c:2157:3: warning: implicit declaration of function 'avahi_service_name_join' [-Wimplicit-function-declaration]
|    avahi_service_name_join(fullName, kDNSServiceMaxDomainName, serviceName,
|    ^
| ippfind.c:2119:9: warning: unused variable 'fullName' [-Wunused-variable]
|    char  fullName[kDNSServiceMaxDomainName];
|          ^
| ippfind.c: At top level:
| ippfind.c:2685:1: warning: 'set_service_uri' defined but not used [-Wunused-function]
|  set_service_uri(ippfind_srv_t *service) /* I - Service */
|  ^
| cc1: warning: unrecognized command line option '-Wno-tautological-compare'
| ../Makedefs:248: recipe for target 'ippfind.o' failed
| make[1]: *** [ippfind.o] Error 1
| make[1]: *** Waiting for unfinished jobs....
| ipptool.c: In function 'with_value':
| ipptool.c:5684:38: warning: logical not is only applied to the left hand side of comparison [-Wlogical-not-parentheses]
|            if (!strcmp(value, "true") == attr->values[i].boolean)
|                                       ^
| ipptool.c: At top level:
| cc1: warning: unrecognized command line option '-Wno-tautological-compare'
| Makefile:31: recipe for target 'all' failed
| make: *** [all] Error 1
| WARNING: exit code 1 from a shell command.
| ERROR: oe_runmake failed
| ERROR: Function failed: do_compile (log file is located at /home/pcp/slicedpie2/wr-core/build-intel-apollolake/tmp/work/core2-64-wrs-linux/cups/2.0.4-r0/temp/log.do_compile.114589)
ERROR: Task 6181 (/home/pcp/slicedpie2/wr-core/layers/oe-core/meta/recipes-extended/cups/cups_2.0.4.bb, do_compile) failed with exit code '1'

**Comment by masselstine** [comment]
This is a duplicate of issue #38 . A fix is forthcoming. For now please add the following to your conf/local.conf to workaround it.

PACKAGE_CONFIG_cups_append += "avahi"

Since this is a duplicate I am going to mark this as closed.
